CI Failing for beta/master due

After merging `release` branch into `beta` (and `beta` into `master`) this morning (which hadn't been done for a while apparently), I introduced some failing tests.

```
not ok 101 Acceptance: ember new verify fixtures addon + npm + !welcome
  AssertionError: expected ".travis.yml" to equal "/home/travis/build/ember-cli/ember-cli/tests/fixtures/addon/npm/.travis.yml"
      at tests/acceptance/new-test.js:423:15
      at Array.forEach (native)
      at Context.<anonymous> (tests/acceptance/new-test.js:421:9)
      at next (native)
      at onFulfilled (node_modules/co/index.js:65:19)
      at tryCatch (node_modules/rsvp/dist/rsvp.js:525:12)
      at invokeCallback (node_modules/rsvp/dist/rsvp.js:538:13)
      at publish (node_modules/rsvp/dist/rsvp.js:508:7)
      at flush (node_modules/rsvp/dist/rsvp.js:2415:5)

not ok 102 Acceptance: ember new verify fixtures addon + yarn + welcome
  AssertionError: expected ".travis.yml" to equal "/home/travis/build/ember-cli/ember-cli/tests/fixtures/addon/yarn/.travis.yml"
      at tests/acceptance/new-test.js:444:15
      at Array.forEach (native)
      at Context.<anonymous> (tests/acceptance/new-test.js:442:9)
      at next (native)
      at onFulfilled (node_modules/co/index.js:65:19)
      at tryCatch (node_modules/rsvp/dist/rsvp.js:525:12)
      at invokeCallback (node_modules/rsvp/dist/rsvp.js:538:13)
      at publish (node_modules/rsvp/dist/rsvp.js:508:7)
      at flush (node_modules/rsvp/dist/rsvp.js:2415:5)
```

--- a/tests/fixtures/addon/npm/.travis.yml
+++ b/tests/fixtures/addon/npm/.travis.yml
@@ -38,4 +38,4 @@ before_install:
 script:
   # Usually, it's ok to finish the test scenario without reverting
   #  to the addon's original dependency state, skipping "cleanup".
-  - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO test --skip-cleanup
+  - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO --skip-cleanup

--- a/tests/fixtures/addon/yarn/.travis.yml
+++ b/tests/fixtures/addon/yarn/.travis.yml
@@ -39,4 +39,4 @@ install:
 script:
   # Usually, it's ok to finish the test scenario without reverting
   #  to the addon's original dependency state, skipping "cleanup".
-  - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO test --skip-cleanup
+  - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO --skip-cleanup

